app/controllers/questions_controller.rb

```diff
@@ -1,6 +1,6 @@
 class QuestionsController < ApplicationController
-  def new
-    @question = Question.new
+  def index
+    @questions = Question.all
```

Comment:
`index` に質問一覧表示と新規投稿の2つがまとまっていますので，`index` には両方必要です。

```suggestion
    @questions = Question.all
    @question = Question.new
```